spec/support/content_item_helper.rb

```diff
@@ -7,4 +7,17 @@ def basic_content_item(title, other_fields: {})
       document_type: "guidance",
     ).merge(other_fields)
   end
+
+  def build_linkable(hash)
```

Comment:
Note that this only returns random content by default. I think that's better than hard coded values, since it requires the tests to be explicit about what they expect the helper method to produce. Perhaps `basic_content_item` should be rewritten to to something like it.